tutormfe/patches/openedx-lms-development-settings

```diff
@@ -19,6 +19,9 @@ MFE_CONFIG = {
     "STUDIO_BASE_URL": "http://{{ CMS_HOST }}:8001",
     "USER_INFO_COOKIE_NAME": "user-info",
     "ACCESS_TOKEN_COOKIE_NAME": "edx-jwt-cookie-header-payload",
+{% if MFE_ACCOUNT_MFE_APP %}
+    "ACCOUNT_SETTINGS_URL": "{{ "https" if ENABLE_HTTPS else "http" }}://{{ MFE_HOST }}/{{ MFE_ACCOUNT_MFE_APP["name"] }}",
+{%- endif %}
```

Comment:
Can you please port this to the block that was introduced in Palm, below?  (This is why I asked for the rebase.)

Reply:
Below is MFE spesfic settings, but this is needed to apply to all MFEs, or am I missing something